src/test/extras.test.ts

```diff
@@ -133,4 +135,57 @@ for (let scenario of scenarios) {
         t.end();
     });
 
+    t.test(scenario.description + ': copyMyDocsToOtherWorkspace', async (t: any) => {
+        let now = Date.now() * 1000;
+        let sourceStorage = scenario.makeStorage(WORKSPACE);
+        let destStorage = scenario.makeStorage(OTHER_WORKSPACE);
+
+        // scenarios:
+
+        // A lone document
+        t.same(sourceStorage.set(keypair1, { format: FORMAT, path: '/path1', content: 'val1 keypair1', timestamp: now - 60 }), WriteResult.Accepted, 'make doc 1...');
+        t.equal(sourceStorage.getContent('/path1'), 'val1 keypair1');
+
+        // an ephemeral document
+        t.same(sourceStorage.set(keypair1, { format: FORMAT, path: '/path2!', content: 'val2 keypair1', timestamp: now - 10, deleteAfter: now + DAY }), WriteResult.Accepted, 'make ephemeral doc 2');
+        t.equal(sourceStorage.getContent('/path2!'), 'val2 keypair1');
+
+        // check that writes worked as expected
+        t.same(sourceStorage.paths(), ['/path1', '/path2!'], 'paths() are correct');
```

Comment:
This might be overzealous from my side, but maybe it'd be good to add a document from another author and verify that it isn't transferred over to the new storage.

Reply:
I love it.  I'll gladly add this scenario to the test.